static/src/js/components/ErrorBanner/ErrorBanner.jsx

```diff
@@ -16,28 +18,75 @@ import Row from 'react-bootstrap/Row'
  */
 export const ErrorBanner = ({
   dataTestId,
-  message
-}) => (
-  <Row>
-    <Col>
-      <Alert variant="danger">
-        <Alert.Heading>Sorry!</Alert.Heading>
+  message,
+  previousURL
+}) => {
+  const currentURL = window.location.pathname
+  const conceptKeywords = ['/collections/', '/variables/', '/services/', '/tools/']
 
-        <span className="visually-hidden">{' '}</span>
+  // Checks to see that the endpoint is correct,
+  // indicating that graphQL made a call that returned null
+  const graphQLCallReturnsNull = (url) => {
```

Comment:
maybe isConceptUrl(url) -- what is the significant or returns null in the method name?

Reply:
I initially thought it sounded too vague and wanted to be more specific. However this is probably leaning towards being more misleading, I can't confirm that data is null. Though that is the most likely cause. Changed back!